src/DssSpell.t.sol

```diff
@@ -974,48 +1006,128 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // SPELL-SPECIFIC TESTS GO BELOW
+    address            immutable L2_BRIDGE              = base.addr("L2_BASE_TOKEN_BRIDGE");
+    address            immutable L2_BRIDGE_IMP          = base.addr("L2_BRIDGE_IMP");
+    address            immutable L2_GOV_RELAY           = base.addr("L2_GOV_RELAY");
+    address            immutable L2_SPELL               = base.addr("L2_SPELL");
+    address            immutable L2_USDS                = base.addr("L2_USDS");
+    address            immutable L2_SUSDS               = base.addr("L2_SUSDS");
+    address            immutable L2_MESSENGER           = base.addr("L2_MESSENGER");
+    address            immutable SUSDS                  = addr.addr("SUSDS");
+    address            immutable L1_BRIDGE              = addr.addr("BASE_TOKEN_BRIDGE");
+    address            immutable L1_GOV_RELAY           = addr.addr("BASE_GOV_RELAY");
+    address            immutable L1_ESCROW              = addr.addr("BASE_ESCROW");
+    address            immutable L1_BRIDGE_IMP          = addr.addr("BASE_TOKEN_BRIDGE_IMP");
+    address            constant  L1_MESSENGER           = 0x866E82a600A1414e583f7F13623F1aC5d58b0Afa;
```

Comment:
nit, I think we can define these as entity inline (eg. `address immutable L2_BRIDGE = L2TokenBridgeLike(base.addr("L2_BASE_TOKEN_BRIDGE"));`
and where we use address use `address(L2_BRIDGE)` to reduce some redundant code